### DISCUSSION TITLE
How to install NordVPN via zinit

### DISCUSSION THREAD

**Discussion by noraworld** [zdharma-continuum/zinit#341] [Q&A]
I’m not sure how to install a command via an installation script such as NordVPN.

To install a NordVPN command, according to [NordVPN support](https://support.nordvpn.com/Connectivity/Linux/1325531132/Installing-and-using-NordVPN-on-Debian-Ubuntu-Raspberry-Pi-Elementary-OS-and-Linux-Mint.htm), all I have to do is type the following command.

```shell
sh <(curl -sSf https://downloads.nordcdn.com/apps/linux/install.sh)
```

But I’d like to manage this via zinit, instead of typing the above command manually.

I’ve tried the following configuration, only fail to exit a terminal every time.

```shell
zinit snippet https://downloads.nordcdn.com/apps/linux/install.sh
```

Thank you in advance.

**Comment by vladdoster** [comment]
What is the output of

```
echo "OS: ${OSTYPE:-N/A} | Vendor: ${VENDOR:-N/A} | Machine: ${MACHTYPE:-N/A} | CPU: ${CPUTYPE:-N/A} | Processor: ${$(uname -p 2>/dev/null):-N/A} | Hardware: ${$(uname -m 2>/dev/null):-N/A}"
```

**Reply by noraworld** [reply]
The output is:

```
OS: linux-gnu | Vendor: unknown | Machine: aarch64 | CPU: aarch64 | Processor: aarch64 | Hardware: aarch64
```

**Comment by vladdoster** [comment]
try this and we can work from there

```zsh
zi light zdharma-continuum/zinit-annex-readurl
zi for \
    as'readurl|null' \
    dlink'nordvpn_%VERSION%_arm64.deb' \
    extract \
    id-as'nord-vpn' \
    nocompile \
  "https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/"
```

**Reply by vladdoster** [reply]
@noraworld Any update on my suggestion?
